platform/ios/src/UIViewController+MGLAdditions.m

```diff
@@ -0,0 +1,22 @@
+#import "UIViewController+MGLAdditions.h"
+
+@implementation UIViewController (MGLAdditions)
+
++ (UIViewController *)getTopViewControllerFrom:(UIViewController *)viewController
```

Comment:
> this can indeed be a class method since it doesn't depend on any state

That’s true as far as the current implementation goes. However, it would be possible to express this method in a different way, so that a UIViewController knows how to return its topmost containing view controller: `-topmostViewController` or perhaps `-rootViewController`.